cime_config/machines/config_batch.xml

```diff
@@ -553,6 +553,7 @@
 
     <batch_system MACH="aurora" type="pbspro">
       <batch_submit>/lus/flare/projects/E3SM_Dec/tools/qsub/throttle</batch_submit>
+      <jobid_pattern>(\d+)\.aurora-pbs</jobid_pattern>
```

Comment:
Occasionally, job submissions' `stdout` returns more output beyond a job's id: e.g.
```
ERROR: Couldn't match jobid_pattern '^(\d+)' within submit output:
 'auth: error returned: 15007
auth: Failed to receive auth token
No Permission.
qstat: cannot connect to server aurora-pbs-0001.hostmgmt.cm.aurora.alcf.anl.gov (errno=15007)
5435693.aurora-pbs-0001.hostmgmt.cm.aurora.alcf.anl.gov'
```
This regex update is to extract job-id from such longer strings to let CIME continue with its job stages.